### ISSUE TITLE
[Icon Carbon React]

### ISSUE THREAD

**Issue by alexanderbedoyahincapie073-ctrl** [carbon-design-system/carbon#21032]
## Environment

> Operating system

> Browser

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by laurenmrice** [comment]
@alexanderbedoyahincapie073-ctrl Could you please explain what icon bug you are experiencing with Carbon React and fill out the above Issue template?

**Comment by Kritvi-bhatia17** [comment]
Since there has been no recent activity, @alexanderbedoyahincapie073-ctrl, please re-open this issue if the problem persists.
